docs/DeveloperGuide.md

```diff
@@ -307,26 +306,26 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * 2a1. TAB displays warning message to user (that there are no students yet)
 
       Use case ends
-* 2b. TAB detects that an identical lab tag already exists.
-    * 2b1. TAB displays error message (that tag already exists)
+* 2b. TAB detects that an identical lab already exists.
+    * 2b1. TAB displays error message (that lab already exists)
 
       Use case ends
 
 **Use case: Filter Students**
 
 **MSS**
 
-1.  TA requests to filter students by a tag 
+1.  TA requests to filter students by a specific lab 
```

Comment:
I'm not sure about this one. The tutor might want to be able to filter students on other tags, not only labs

Reply:
Referring to our User Guide for v1.2, I think we only planned to have only have a lab as a tag for now. But if we do plan on adding more tag categories in the future.